orion-ui/src/pages/FlowRun.vue

```diff
@@ -84,7 +84,7 @@
   const flowRunId = useRouteParam('id')
 
   const tabs = computed(() => {
-    const values = ['Logs', 'Task Runs', 'Sub Flow Runs', 'Parameters']
+    const values = ['Logs', 'Task Runs', 'SubFlow Runs', 'Parameters']
```

Comment:
Should this be Subflow runs, with a lowercase F and R? Are we using sentence case for Task Runs also?

We might also consider putting each element on its own line as well, to make diffs easier to read

Reply:
good points, both are implemented